### ISSUE TITLE
SUMIFS missing and old code buggy

### ISSUE THREAD

**Issue by ZohraS** [audreyt/ethercalc#362]
Hi guys,

I've tried using SUMIFS as a formula but didn't get far. After checking the code, I realized that the SUMIFS code added here #108 wasn't included in the latest code. So I added it into my local copy. 

All seemed fine, but as soon as I add the formula with a second arg, it just ignores it, no matter what the second argument is.

I'm going to look into the code to mess with it a bit, maybe I can get it working, but otherwise, thought you should know that the sumifs function is missing.

**Comment by eddyparkinson** [comment]
**Merge error**
That would be my fault :( 
It looks like I "removed" SUMIFS and RAND by mistake. 
I was new to GIT merge when I did it, and it looks like I messed up the merge.
@ZohraS  I will re-add it and check the merge again. - or do you plan to add it?

**DSUM**
SUMIFS is a lot like DSUM - they can do more or less the same thing.

**Comment by ZohraS** [comment]
Hey Eddy,

Thanks so much for getting back to us. Even when I added the original code, this SUMIFS gave me 0, even though it should be a value.

I don't even know anymore, I'm getting super confused. This is the formula we're using:

=SUMIFS(testsumifs.5!C3:C100,testsumifs.5!D3:D100,testsumifs.4!N6,testsumifs.5!E3:E100,O6,testsumifs.5!B3:B100,'<42319')

We've tried with and without the single quotes, but it seems that the second arguments are getting ignored completely, we tried with just one arg pair (arg + criteria) and that works fine... 

I'd rather not touch the git, as I'm trying to find a solution for this problem.

Thanks in advance!
